olca-app-html/src/home/messages.ts

```diff
@@ -6,252 +6,352 @@ const messages: Record<string, Record<string, string>> = {
         "title2": "For modeling the life cycle of things.",
         "title3": "Licenced under the Mozilla Public Licence 2.0.",
         "title4": "Created and maintained since 2006 by GreenDelta, Berlin",
-        "whatsNew.title": "What is new in openLCA",
-        "whatsNew.text": "Visit <a id=\"news\">openlca.org/new</a> to find out about improvements and changes in the most recent version of openLCA.",
-        "whatsNew.link": "",
-        "gettingStarted.title": "Getting started",
-        "gettingStarted.text": "Manuals and case studies are available for free on <a id=\"manuals\">openLCA.org/learning</a> and the <a id=\"channel\">openLCA YouTube channel</a>.",
-        "gettingStarted.link": "You can also book a professional <a id=\"trainings\">openLCA training</a>.",
-        "manuals.title": "Manuals, case studies and data",
-        "manuals.text": "Create a new database or restore an existing LCA database from a *.zolca file. You may also want to import an LCIA method package or other data into your database. Visit <a id=\"nexus\">openLCA Nexus</a> and <a id=\"dataopenlca\">data.openLCA.org</a> to download LCA data.",
-        "manuals.link": "For instructions, watch <a id=\"video\">this video</a> on the openLCA YouTube channel.",
-        "community.title": "Community",
-        "community.text": "openLCA is an open source software by <a id=\"greendelta\">GreenDelta</a>.",
-        "community.link": "Since its first release, an active openLCA community has emerged with international partners in various countries and tens of thousands of users. Join and get engaged on our public support platform for openLCA and the LCA Collaboration Server <a id=\"forum\">ask.openLCA.org</a>. Follow openLCA on <a id=\"linkedin\">LinkedIn</a> or <a id=\"twitter\">Twitter</a> for news and updates.",
-        "experts.title": "Make the most out of openLCA",
-        "experts.text": "GreenDelta, the creator of openLCA, offers openLCA <a id=\"services\">support services</a> via the GreenDelta helpdesk.",
-        "experts.link": "The <a id=\"lcacs\">LCA Collaboration Server</a> helps you to get the most out of working with openLCA in your team. If you do not want to run an LCA Collaboration Server yourself, GreenDelta offers you <a id=\"hosting\">hosting packages</a>."
+        "firstSteps.title": "1st steps in openLCA",
+        "firstSteps.text": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
```

Comment:
Why did you change the text for a Lorem Ipsum?

Reply:
I do not have the final draft of the text.  This was the proposed design for now. 
[olca2_welcome.docx](https://github.com/GreenDelta/olca-app/files/11200693/olca2_welcome.docx)

Reply:
OK I did not know about that. Let's keep the current texts for the time being. I think it is better to have these on `master`rather than Lorem Ipsum as we make releases that need to be clean.

Reply:
Yes even I thought so . Actually the headings are different than earlier so will be the text.  I will update the  new text before merging to the main . I will not merge with this Lorem Ipsum :) .